Astro Release Notes (June 9th) (#792)
astronomer/docs: astro/cli/astro-deployment-update.md

astro deployment update <deployment-id> <flags>
```

:::tip

> To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:

Reviewer: ```suggestion
To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md) and make the variables accessible to the Astro CLI, set the following OS-level environment variables:
```

Author: See previous comment

- `ASTRONOMER_KEY_ID`
- `ASTRONOMER_KEY_SECRET`

> After setting the variables, this command works for a Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.

Reviewer: ```suggestion
After setting the variables, this command works for a Deployment you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
